<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab_Project</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2b2b2b;   /* macOS dark gray */
color: #f0f0f0;              /* slightly off-white text */
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1e1e1e;
color: #ffffff;
border: 1px solid #555;
border-radius: 4px;
padding: 3px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Resistor 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Resistor 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3a3a3a;
color: #ffffff;
border-radius: 6px;
padding: 5px;
</string>
    </property>
    <property name="text">
     <string>Simulate</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1e1e1e;
color: #ffffff;
border: 1px solid #555;
border-radius: 4px;
padding: 3px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Voltage</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>431</width>
      <height>171</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1e1e1e;
color: #ffffff;
border: 1px solid #555;
border-radius: 4px;
padding: 3px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Resistor 1 Tolerance</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Resistor 2 Tolerance</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>36</height>
     </size>
    </property>
    <property name="toolTip">
     <string>Template combo for Section column (Apple-style rose gradient)</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5,
        stop:0 rgba(255,140,90,255),
        stop:1 rgba(255,57,0,255));
    color: white;
    padding-left: 12px;
    padding-right: 30px;
    border-radius: 15px;
    font-size: 14px;
    min-height: 36px;
}

/* Arrow Area */
QComboBox::drop-down {
    width: 26px;
    border: none;
}

/* The arrow icon */
QComboBox::down-arrow {
    image: url(&quot;C:/Users/ibrah/Desktop/New folder 111/icons_exported/iphone_rose.png&quot;);
    width: 25px;
    height: 25px;
}
</string>
    </property>
    <item>
     <property name="text">
      <string>Select Tolerance</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>±10%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>±5%</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>36</height>
     </size>
    </property>
    <property name="toolTip">
     <string>Template combo for Section column (Apple-style rose gradient)</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5,
        stop:0 rgba(255,140,90,255),
        stop:1 rgba(255,57,0,255));
    color: white;
    padding-left: 12px;
    padding-right: 30px;
    border-radius: 15px;
    font-size: 14px;
    min-height: 36px;
}

/* Arrow Area */
QComboBox::drop-down {
    width: 26px;
    border: none;
}

/* The arrow icon */
QComboBox::down-arrow {
    image: url(&quot;C:/Users/ibrah/Desktop/New folder 111/icons_exported/iphone_rose.png&quot;);
    width: 25px;
    height: 25px;
}
</string>
    </property>
    <item>
     <property name="text">
      <string>Select Tolerance</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>±10%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>±5%</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
